db/citations.yml

```diff
@@ -10092,3 +10092,42 @@ zhu;gca11:
   wwwemail: ''
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/12zhugca.pdf
   year: '2012'
+lo;nc20:
+  author:
```

Comment:
please create an ``ackno`` field and copy-paste the acknowledgements section.

Reply:
Here is a problem about the acknowledgement. It ackowledge the mrsec but actually it shouldn't. We discussed about this mistake before but it seemed that no addendum was published for it. The only addendum published for this paper is about the authors' name and institution.